src/AddBlogTitle.jsx

```diff
@@ -0,0 +1,17 @@
+export default function AddBlogTitle(props) {
+
+    function onSubmit(event) {
+        event.preventDefault()
+        const newBlogTitle = event.target[0].value
+        props.setNewTitle(newBlogTitle)
```

Comment:
setNewTitle is a callback handler and `setNewTitle(newBlogTitle)` is lifting the state of the input field value out of the `AddBlogTitle` component.